constraint/pkg/apis/templates/init.go

```diff
@@ -0,0 +1,6 @@
+package templates
+
+func init() {
```

Comment:
The problem wasn't the inability to have multiple `init()` calls, but rather that `initializeCTSchemaMap()` uses the `Scheme` that's created in `initializeScheme()`.  As init functions are run [alphabetically](https://medium.com/@markbates/go-init-order-dafa89fcef22#2ad9:~:text=As%20you%20can%20see%20the%20init%20functions%20are%20loaded%20in%20order%20by%20the%20filename.), this caused a null-pointer exception.

Rather than fix this through naming, I figured the best way to was to call them in an intentional order like I did in this `init.go` file.  I'll add a comment to `init.go` explaining that choice.

Or, perhaps there's a pattern that would better suit this use-case? @willbeason

Reply:
Is there a reason the contents of those two methods can't be in the same function? `initializeScheme` is rather short.